### ISSUE TITLE
pyzeebe installation fails on Mac OS X Catalina (10.15.7) due to dependency on grpcio 1.32.0

### ISSUE THREAD

**Issue by StevenDas-Tomo** [JonatanMartens/pyzeebe#107]
**Describe the bug**

`pip install pyzeebe==2.2.2` on Mac OS X Catalina (10.15.7) fails due to an incompatibility between grpcio 1.32.0 and GCC:

https://github.com/grpc/grpc/issues/25060#issuecomment-758270979

Interestingly, `pip` installs the latest stable version of `grpcio` (1.34.0) without issue.

I'm using the following dependencies:

- `pip`: 20.2.3
- `python`: 3.9.1
- `gcc`:

```
Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/c++/4.2.1
Apple clang version 12.0.0 (clang-1200.0.32.28)
Target: x86_64-apple-darwin19.6.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
```

**To Reproduce**
1. Run `pip install pyzeebe==2.2.2` on Mac OS X Catalina (10.15.7) with the exact dependencies specified above.

**Expected behavior**
Installation via `pip` should succeed without error.

**Additional context**
I notice that the `grpcio` dependency was removed 25 days ago:

https://github.com/JonatanMartens/pyzeebe/pull/91/files

Can we please release a new version of `pyzeebe` which includes this commit? That should unblock us.

Thank you so much for taking the time to maintain this useful library!

**Comment by StevenDas-Tomo** [comment]
Unfortunately, I now see that `zeebe-grpc` also depends on version 1.32.0 of `grpcio`:

https://gitlab.com/stephane.ludwig/zeebe_python_grpc/-/blob/master/requirements_dev.txt

I'll raise the same issue over there.

**Comment by StevenDas-Tomo** [comment]
Actually, it appears that the latest version of `zeebe-grpc` (0.25.1.0) depends on version 1.33.2 of `grpc`:

https://gitlab.com/stephane.ludwig/zeebe_python_grpc/-/blob/master/setup.py#L17

I had no trouble `pip install`ing this version of `zeebe-grpc` onto my machine.

So it appears that pinning to version 0.25.1.0 of `zeebe-grpc` should fix this issue.

**Comment by JonatanMartens** [comment]
Hey @StevenDas-Tomo, thanks for bringing this to our attention.
Version 2.2.3 was just released, so I'm closing this.
Please feel free to reopen this issue if you encounter any more problems.

**Comment by kbakk** [comment]
I managed to reproduce this with `pyzeebe==2.2.2` + Python 3.9 - not 3.8 or 3.7 however (interesting). It's resolved with `pyzeebe==2.2.3`. 🙂 

I think this issue is related: https://gitlab.com/stephane.ludwig/zeebe_python_grpc/-/issues/4 - as it will allow the user to specify the grpc version (or simply resolve to latest, which wouldn't have caused this). I gave it a thumb-up, hopefully it will be prioritized soon.
